src/panels/config/entities/entity-registry-settings.ts

```diff
@@ -276,7 +276,6 @@ export class EntityRegistrySettings extends SubscribeMixin(LitElement) {
                 @selected=${this._deviceClassChanged}
                 @closed=${stopPropagation}
               >
-                <mwc-list-item></mwc-list-item>
```

Comment:
hmm this got carried over it seems. Can you add this back.

Reply:
Whoops!